functions-methods.js

```diff
@@ -33,4 +51,19 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email) {
+    if (email.indexOf("@") === -1) {
+        return false;
+    }
+    if (email.indexOf(",") !== -1) {
+        return false;
+    }
+    if (email.endsWith(".")) {
+        return false;
+    }
+    return true;
```

Comment:
Dit kun je nog iets versimpelen
```suggestion
    return email.includes("@") && !email.includes(",") && !email.endsWith(".");
```